Source/WebCore/dom/ElementRareData.h

```diff
@@ -95,6 +95,9 @@ class ElementRareData : public NodeRareData {
     NamedNodeMap* attributeMap() const { return m_attributeMap.get(); }
     void setAttributeMap(std::unique_ptr<NamedNodeMap>&& attributeMap) { m_attributeMap = WTFMove(attributeMap); }
 
+    String userInfo() const { return m_userInfo; }
+    void setUserInfo(String& userInfo) { m_userInfo = userInfo; }
```

Comment:
Can we use move semantics for this setter?

```suggestion
    void setUserInfo(String&& userInfo) { m_userInfo = WTFMove(userInfo); }
```

Reply:
Sure.